site/zenodo_rdm/assets/semantic-ui/js/zenodo_rdm/src/support/components/SupportForm.js

```diff
@@ -0,0 +1,290 @@
+import { PropTypes } from "prop-types";
+import React, { Component } from "react";
+import { Formik, Form } from "formik";
+import { Form as SemanticForm, Button, Modal } from "semantic-ui-react";
+import { TextField, TextAreaField, ToggleField } from "react-invenio-forms";
+import CategoryDropdown from "./CategoryDropdown";
+import * as Yup from "yup";
+import axios from "axios";
+import Dropzone from "react-dropzone";
+import FileTable from "./FileTable";
+import _remove from "lodash/remove";
+import { humanReadableBytes } from "react-invenio-deposit";
+
+// TODO translations
+// TODO implement error notification
+// TODO implement recaptcha
+// TODO required fields are highlighted by the component and not Formik
+
+const apiConfig = {
+    withCredentials: true,
+    xsrfCookieName: "csrftoken",
+    xsrfHeaderName: "X-CSRFToken",
+    headers: {
+        "Accept": "application/json",
+        'Content-Type': 'multipart/form-data',
+    },
+};
+
+const http = axios.create(apiConfig);
+
+function formikToFormData(values) {
+    const formData = new FormData();
+    // Append each key/value pair to formData
+    Object.keys(values).forEach((key) => {
+        let currentValue = values[key];
+        if (Array.isArray(currentValue)) {
+            currentValue.forEach((val) => {
+                formData.append(key, val);
+            })
+        } else {
+            formData.append(key, currentValue);
+        }
+    })
+
+    return formData;
+}
+
+
+const FileUploader = ({ dropzoneParams, maxFileSize, name, currentFiles, handleDelete }) => {
+    return (
+        <Dropzone {...dropzoneParams}>
+            {({ getRootProps, getInputProps }) => {
+                const inpProps = {
+                    ...getInputProps(),
+                    // Display the dropzone input, otherwise it's hidden by default.
+                    style: {
+                        'display': 'block'
```

Comment:
`react-dropzone` has an `input` element that is hidden. 

In other parts of invenio we keep it that way and re-implement it with either a button or a box for drag and drop. Here I went for a simpler solution, might not be the best though.